QUARKUS-960.md

```diff
@@ -0,0 +1,29 @@
+# Quarkus-960 - Improved getting started experience with CLI
+JIRA link: https://issues.redhat.com/browse/QUARKUS-960
+
+There was an existing CLI tool in Quarkus that has been fully reworked as part of QUARKUS-960. 
+
+So, the goal is to verify proper functionality of the new Quarkus CLI tool.
+
+## Scope of the testing
+
+The Quarkus CLI is compatible with Maven, Gradle and jbang tools, however RHBQ is only focused on Maven, so Gradle and Jbang will be covered as part of the test coverage in upstream.
+
+Test development will focus on the supported features scoped for RHBQ 2.2:
+- Create applications (with a set of extensions)
+- Create extensions
+- Build existing Quarkus applications on JVM
+- Build existing Quarkus applications on Native (also using container configuration and builder images)
+- Run Quarkus applications on DEV mode
+- Run Quarkus applications on Remote Dev
+
+### Impact on testsuites and testing automation:
+ - Enhance Quarkus QE test framework to cover all the features that are part of the scope
+ - Add test coverage into the Quarkus QE test suite only for baremetal
+ - Ensure this coverage works both in JVM and NATIVE mode 
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+ - Focus on exploratory testing of the feature
+ - Ensure good user experience and simplicity of use
+
```

Comment:
Add experiments on Windows as that can get interesting

Reply:
Added as part of the manual verification too